manifests/d/Diskuv/OCaml/1.0.0.0/Diskuv.OCaml.installer.yaml

```
@@ -0,0 +1,70 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+# Follow: https://docs.microsoft.com/en-us/windows/package-manager/package/manifest?tabs=minschema%2Cversion-example#tips-and-best-practices
+PackageIdentifier: "Diskuv.OCaml"
+PackageVersion: "1.0.0.0"
+Scope: user
+UpgradeBehavior: install
+Commands:
+  # /bin
+  - opam
+  - opam-installer
+  - opam-putenv
+  # /usr/bin (max 16 Commands so took only the most commonly used)
+  - dune
+  - ocaml
+  - ocamlc
+  - ocamldebug
+  - ocamldoc
+  - ocamlfind
+  - ocamlformat
+  - ocamllsp
+  - ocamlopt
+  - ocamlrun
+  - ocp-indent
+  - utop
+  - with-dkml
+Platform:
+  - "Windows.Desktop"
+# Minimum OS comes from https://github.com/diskuv/dkml-runtime-distribution/blob/ff7e6bcbf3dbdb37e64b20a57e2dea0fa9d27ed5/src/windows/Machine/Machine.psm1#L14:
+# > $Windows10SdkVer = "18362"
+MinimumOSVersion: "10.0.18362.0"
+InstallerType: exe
+InstallModes:
+  - silent
+  - interactive
+Dependencies:
+  PackageDependencies:
+    # ---- Visual Studio Build Tools ----
+
+    # Minimum comes from https://github.com/diskuv/dkml-runtime-distribution/blob/ff7e6bcbf3dbdb37e64b20a57e2dea0fa9d27ed5/src/windows/Machine/Machine.psm1#L94-L105:
+    # > $VcStudioVcToolsMajorVer = 16
+    # > $VcStudioVcToolsMinorVer = 6
+    - PackageIdentifier: Microsoft.VisualStudio.2019.BuildTools
+      MinimumVersion: 16.6.0
+    
+    # ---- Git ----
+    # Minimum comes from https://github.com/diskuv/dkml-component-ocamlcompiler/blob/66af047444345a012213980269faffdf8ea83fb0/assets/staging-files/win32/setup-userprofile.ps1#L602-605
+    - PackageIdentifier: Git.Git
+      MinimumVersion: 2.34.0
+Installers:
+  - Architecture: "x64"
+
+    # TODO: InstallerUrl/InstallerSha256/ReleaseDate need to be updated by contributors/sign.sh!
+    InstallerUrl: "https://github.com/diskuv/dkml-installer-ocaml/releases/download/v1.0.0/setup-diskuv-ocaml-windows_x86_64-1.0.0.exe"
+    InstallerSha256: 32f010577a791d5595760cbb5136bd60049ffd37c8d44e68e7a8729319670a48
+    ReleaseDate: 2022-08-08
+
+    InstallerSwitches:
+      Silent: --ci --quiet --color=never
+      SilentWithProgress: --ci
+      Interactive: --color=always
+      InstallLocation: --prefix "<INSTALLPATH>"
+
+    AppsAndFeaturesEntries:
+      # Keep in sync with installer\src\private_common.ml which is used by
+      # https://github.com/diskuv/dkml-install-api/blob/main/package/console/common/Windows_registry.ml
+      - Publisher: Diskuv, Inc.
+        DisplayName: Diskuv OCaml
+        DisplayVersion: 1.0.0.0
```

```suggestion
        DisplayVersion: dev
```
This must exactly match what is written into control panel